### ISSUE TITLE
Tied maximum values of alpha MLE

### ISSUE THREAD

**Issue by mpajares-murgo** [kpmainali/CooccurrenceAffinity#4]
Dear colleague, 

We have read your recent paper in Science Advances and we are interested in applying MLE alpha to analyze the similarity of fungal communities associated with different plant species. We have used your R script, and it produces a lot of repeated estimates that correspond to the maximum observed value. We have noticed that it also occurs in the worked example in your web page. This is a problem for our analyses because we would like to use MLE alpha values as dependent variable in a GLM, but it is impossible to normalize the distribution and it does not fit any of the standard distribution families. So, we wonder whether the abundance of repeated maximum values is a property of the estimation of alpha MLE and why does it occur. 

Saludos!

Mariona

**Comment by kpmainali** [comment]
Can you give some examples with values that can be supplied to ML.Alpha()? Have you been able to read about scal parameter in the argument description and details? We have a statistical reason for capping it that way. However, I am happy to look into your situation and potentially adjustment the default cap.

**Comment by mpajares-murgo** [comment]
Of course. I attached the binary matrix of plant species and fungal OTUs in this comment. We are interested in obtaining the similarity between plan species in their associated fungal communities. 

[fungal_bin.csv](https://github.com/kpmainali/CooccurrenceAffinity/files/8221914/fungal_bin.csv)

**Comment by kpmainali** [comment]
Hello, sorry about this delay in responding. I can see that pairs like "ACE_MON  ROS_JAE" and "AME_OVA  ROS_JAE" have a tie at 10.00. As we discussed in the original paper (https://www.science.org/doi/full/10.1126/sciadv.abj9204?af=R), we have applied some restriction on MLE of alpha. This defaults to 2*N^2, capped at 10 within the function. This is to prevent some extreme estimates which are already not very meaningful.
